### DISCUSSION TITLE
High-density displays inconsistencies

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#237] [Q&A] [answered]
This is not super important, but I'd like to understand what is going on...

At work I have access to a Windows machine with a high-density and P3D is comming very small. 
I'm using `pixel_density(2)`.

**P3D** (this comes up very small on screen)
![image](https://user-images.githubusercontent.com/3694604/218823567-c8f7bf19-3f5d-485f-a2b6-c9344baefde4.png)

**Regular renderer**
![image](https://user-images.githubusercontent.com/3694604/218824660-1ac0c811-70db-4e17-9f62-c6af2a053d94.png)
Also `pixelDensity(2) is not available for this display`

**Comment by hx2A** [comment, marked as answer]
Yes, that is rather odd. What is the Display Scaling set to? I've noticed when it is set to something other than 100% some things were not quite right. I believe this is a Processing 4 thing that introduced some features for Windows high density displays. We might need to open an issue for this.

**Reply by villares** [reply]
Cool, I think it is set to 250%...

**Reply by hx2A** [reply]
There's supposed to be a way for Processing to detect that setting and adjust accordingly. You shouldn't be required to set the scaling to 100%.

I think there is something that Processing or perhaps the Processing IDE does that py5 is not also doing to detect this setting and adjust. I remember reading about it in the release notes and thinking I should investigate more but there were other more pressing issues at the time.
